lib/posthog/client.rb

```diff
@@ -82,6 +84,7 @@ def clear
     #
     # @option attrs [String] :event Event name
     # @option attrs [Hash] :properties Event properties (optional)
+    # @option attrs [Bool] :send_feature_flags Whether to send feature flags with this event (optional)
```

Comment:
IMO this should be top-level configuration rather than polluting every event.

Reply:
The thing is, we really don't want this to be set globally, as only specific events _should_ need this option (when the event is used in an experiment, and it's the only event used in an experiment).

And even in those cases, I'd expect people who care about performance to _just_ get the single experiment flag and set its properties, rather than getting all flags from `/decide`.

Reply:
add default